### DISCUSSION TITLE
Entity Configuration URL of Relying Party

### DISCUSSION THREAD

**Discussion by lucamontano98** [OpenIDC/mod_auth_openidc#1144] [Q&A]
Good morning,
can you tell me if there is a way to get the Entity Configuration URL of the Relying Party which has been configured? Thanks.

Luca

**Comment by zandbelt** [comment]
I'm not aware of what Entity Configuration URL refers to, it's not part of OpenID Connect

**Comment by lucamontano98** [comment]
Excuse me! With Entity Configuration URL of Relying Party I refer to its metadata URL. I would like to get the metadata of Relying Party using a URL, as done with OpenID Provider inside the Relying Party configuration file in mod_auth_openidc. Is it possible? Thanks.

Luca

**Comment by zandbelt** [comment]
there's no spec for that, at least not in the OpenID Connect core specs
